frontend/src/pages/NFTInfoPage.js

```diff
@@ -0,0 +1,570 @@
+		<Margin height="2px"/>
+
+		<ParagraphTypography style={{ fontWeight: "700", fontSize: "33px", }}>
+			<Image src={ethIcon} alt="ETH" height="0.8em" display="inline"/>
+			{price}
+			<Margin width="31px"/>
+		</ParagraphTypography>
+	</div>);
+
+	let nftInfoAction;
+
+	const hasEditRights = true;  // TODO, only Admin or original Author can edit (but not buy)
+	const isOwner = false;  // TODO, this could be passed as argument from the default component above
+	if (hasEditRights) {
+		nftInfoAction = <NFTInfoEditOptions/>;
+	} else if (price && !isOwner) {
+		nftInfoAction = <NFTInfoPlaceOrderButton/>;
+	}
+
+	return render();
+};
+
+const NFTInfoEditOptions = () => {
+	const unpublishButtonStyle = { height: "36px", width: "113px", fontWeight: "700", fontSize: "14px", background: activeTextColor, };
+	const unpublishPath = "";  // TODO, maybe an onClick event would be better
+	const editItemPath = "";  // TODO
+	const deleteItemPath = "";  // TODO
+
+	const NFTInfoActionButton = ({ children, style, ...props }) => (
+		<RoundedEdgesButton style={{ fontWeight: "700", fontSize: "14px", color: "#C4C4C4", ...style, }} component={Link} {...props}>
+			{children}
+		</RoundedEdgesButton>
+	);
+
+	const actions = (<div style={{ display: "flex", alignItems: "baseline" }}>
+		<RoundedEdgesButton style={unpublishButtonStyle} component={Link} to={unpublishPath}>
+			Unpublish
+		</RoundedEdgesButton>
+
+		<Margin width="23px"/>
+
+		<NFTInfoActionButton to={editItemPath}>
+			Edit Item
+		</NFTInfoActionButton>
+
+		<Margin width="20px"/>
+
+		<NFTInfoActionButton to={deleteItemPath}>
+			Delete
+		</NFTInfoActionButton>
+	</div>);
+
+	return (<>
+		<div style={{ fontSize: "33px", }}>
+			<NFTInfoBuyingLabel><br/></NFTInfoBuyingLabel>
+
+			<Margin height="10px"/>
+
+			<Margin width="0px" heigth="1em" borderMargin="43px"/>
+		</div>
+
+		<div>
+			<NFTInfoBuyingLabel>Actions:</NFTInfoBuyingLabel>
+
+			<Margin height="10px"/>
+
+			{actions}
+		</div>
+	</>);
+};
+
+const NFTInfoPlaceOrderButton = () => {
+	const buyButtonStyle = { padding: "0px", height: "57px", width: "192px", fontWeight: "700", fontSize: "18px", background: activeTextColor, };
+	const buyPath = "";  // TODO, could be that we need onClick event instead
+	
+	return (<div>
+		<NFTInfoBuyingLabel><br/></NFTInfoBuyingLabel>
+
+		<RoundedEdgesButton style={buyButtonStyle} component={Link} to={buyPath}>
+			Place Order
+		</RoundedEdgesButton>
+	</div>);
+};
+
+const NFTDescription = () => {
+	const nftDescriptionContent = (<>
+		Subtitle or description, can be in two rows.
+		Or even longer description can be here or somewhere else.
+		Subtitle or description, can be in two rows.
+		Or even longer description can be here or somewhere else.<br/><br/>
+
+		Subtitle or description, can be in two rows.<br/><br/>
+
+		Or even longer description can be here or somewhere else.
+		Subtitle or description, can be in two rows.
+		Or even longer description can be here.
```

Comment:
For now we should add some text here where we just fill in some infromation from the drop as we do not have a way to set a custom description atm

Reply:
I put your advice in a TODO comment there.